examples/script_nifti.py

```diff
@@ -1,10 +1,37 @@
+import os
+import pathlib
+import shutil
+import sys
 from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
```

Comment:
Either write

```suggestion
import urllib.request
```

or

```suggestion
from urllib.request import urlopen
```

I know numpy encourages this "well known import alias" style with its "import numpy as np" but I think it's wrong. If they want to be called "np" they should rename themselves in their package, or provide a side-package named "np". Aliasing makes the code you call look different than the code as written in the library which makes reading it difficult and non-standard.